apps/pie-storybook/stories/pie-button.stories.ts

```diff
@@ -40,7 +40,7 @@ const buttonStoryMeta: ButtonStoryMeta = {
             control: 'select',
             options: types,
             defaultValue: {
-                summary: 'submit',
+                summary: 'button',
```

Comment:
QQ – The default for this type in the actual component is `submit` as that matches what most browsers default to when rendering a button.

Should we keep the default here to be `submit` to match the component default?

Reply:
the default has been changed in this PR to button. I can change it back in a follow up if you prefer